k8s/mgmt-sandbox/cluster-00/traefik/traefik.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: traefik
+  namespace: admin
+  annotations:
+    #flux.weave.works/ignore: "true"
+    flux.weave.works/automated: "false"
+spec:
+  releaseName: traefik
+  chart:
+    repository: https://kubernetes-charts.storage.googleapis.com/
+    name: traefik
+    version: 1.61.1
+  values:
+    imageTag: 1.7.9
+    loadBalancerIP: 10.10.9.250
+    resources:
+      requests:
+        cpu: 100m
+        memory: 20Mi
+      limits:
+        cpu: 1000m
+        memory: 300Mi
+    service:
+      annotations:
+        service.beta.kubernetes.io/azure-load-balancer-internal: "true"
+    rbac:
+      enabled: true
+    dashboard:
+      enabled: true
+      domain: traefik00.service.core-compute-mgmt-sandbox.internal
```

Comment:
management doesn't use these domains (no ASE network and that's what core-compute is)
historically we've used
.platform.hmcts.net with a sandbox prefix in the domain name
there's also the option of doing .sandbox.platform.hmcts.net